### PR TITLE
Create gui on rewrite

### PR DESCRIPTION
**Pull request by IanDuncanT**

### CODE REVIEW

**Comment by IanDuncanT** [comment]
The failed test will work once merged. This is because the scripts directory is not cloned when there is no file in it. The master branch has files in it.

File: installer.py [hunk 1]
```diff
@@ -1,3 +1,25 @@
+def full_install(package):
+    import configparser
```
Comment (Wabri) [inline installer.py:2]: I didn't know this package, it's wonderfull

**Review by Wabri — COMMENTED** [review]
Everything seems ok.
Could you show me the gui result?

File: installer.py (hunk 1 above)
Comment (IanDuncanT) [reply installer.py:2]: It is very easy to use for the config.

**Comment by IanDuncanT** [comment]
screen shot 2018-12-29 at 17 01 44

**Comment by IanDuncanT** [comment]
This works like the current yapi gui. The 0 in the result is the exit code of the process for installing. I need to figure out how to intercept terminal output instead of the exit code which is returned by subprocess.call()

**Pull request merged**